tests/core/test_actions.py

```diff
@@ -363,10 +370,12 @@ async def test_action_utter_retrieved_response(
     default_tracker.latest_message = UserMessage(
         "Who are you?",
         parse_data={
-            "response_selector": {
+            RESPONSE_SELECTOR_PROPERTY_NAME: {
```

Comment:
optional: I am not sure we should extensively use constants in the tests:
- it makes them less readable
- it tightly couples the test to the implementation: the test simulates an API user, if we use the same constants here as we do in the code, it is less obvious that changing that constant breaks that API (as the test will still pass)

Reply:
Makes sense! I'll change them back to the actual strings.